src/fides/api/ops/service/connectors/consent_email_connector.py

```diff
@@ -119,15 +185,49 @@ def batch_email_send(self, privacy_requests: Query) -> None:
             filtered_user_identities: Dict[
                 str, Any
             ] = filter_user_identities_for_connector(self.config, user_identities)
-            if filtered_user_identities and privacy_request.consent_preferences:
+
+            # Backwards-compatible consent preferences for old workflow
+            consent_preference_schemas: List[Consent] = [
+                Consent(**pref) for pref in privacy_request.consent_preferences or []
+            ]
+
+            # Privacy preferences for new workflow
+            filtered_privacy_preference_records: List[
+                PrivacyPreferenceHistory
+            ] = filter_privacy_preferences_for_propagation(
+                self.configuration.system, privacy_request.privacy_preferences
+            )
+            filtered_privacy_request_schemas: List[
+                MinimalPrivacyPreferenceHistorySchema
+            ] = [
+                MinimalPrivacyPreferenceHistorySchema.from_orm(privacy_pref)
+                for privacy_pref in filtered_privacy_preference_records
+            ]
+
+            if filtered_user_identities and (
+                consent_preference_schemas or filtered_privacy_preference_records
+            ):
                 batched_consent_preferences.append(
                     ConsentPreferencesByUser(
                         identities=filtered_user_identities,
-                        consent_preferences=privacy_request.consent_preferences,
+                        consent_preferences=consent_preference_schemas,
+                        privacy_preferences=filtered_privacy_request_schemas,
                     )
                 )
             else:
                 skipped_privacy_requests.append(privacy_request.id)
+                ExecutionLog.create(
+                    db=db,
+                    data={
+                        "connection_key": self.configuration.key,
+                        "dataset_name": self.configuration.name,
+                        "collection_name": self.configuration.name,
+                        "privacy_request_id": privacy_request.id,
+                        "action_type": ActionType.consent,
+                        "status": ExecutionLogStatus.skipped,
+                        "message": f"Consent email skipped for '{self.configuration.name}'",
+                    },
+                )
```

Comment:
For better record-keeping, we were just not adding any logs for email connectors if the request wasn't fired. Now adding a skipped log instead.  Actually, I forgot I made this a method above, so I'll call `self.add_skipped_log(db, privacy_request)` here